backendRestDraft/src/main/java/de/pretrendr/jobs/ScheduledTasks.java

```diff
@@ -0,0 +1,46 @@
+package de.pretrendr.jobs;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Component
+public class ScheduledTasks  implements ApplicationRunner {
+	private static final Logger LOGGER = LoggerFactory.getLogger(ScheduledTasks.class);
```

Comment:
Annotate the class with `@Slf4j`, this will autoinject a Logger called `log` into the class.

Reply:
changed